slider/slider.css

```diff
@@ -0,0 +1,200 @@
+@import "../resetcss.css";
+
+
+@media screen and (max-width: 768px){
+ .slider-box{
+    max-width: 35rem;   
+    height: auto;
+ }    
+ .show-dimension{
+   border: 1px solid black;
+   
+}
+.show-dimension img{
+   width: 100%; 
+   height: auto;
+}
+}
+
+/* ------------------------------------------------------------------------------------------------------------------- */
+
+.skin{
+    padding: 0.5rem 0.5rem 0 0.1rem;
+    
+}
+.card-skin{
+    margin: 1rem;
+    width: 35rem;
+}
+
+a{
+  font-size: 2rem;
+  text-decoration: none;
+  color: black;
+}
+
+  article a{
+      display: block;
+      margin-top: 1rem;
+      padding-left: 2rem;
+      
+  
+  }
+  article{
+      display: flex;
+      flex-direction: column;
+      gap: 1rem;
+      /* margin : 2rem 0 8rem 0; */
+  }
```

Comment:
I see repeated styles here. Saw similar styles in modal.css. Maybe you could combine them?